helpers/extension/driver.js

```diff
@@ -30,15 +31,21 @@ class ExtensionProtocol extends ChromeProtocol {
     chrome.debugger.onEvent.addListener(this._onEvent.bind(this));
   }
 
-  connect() {
+  connect(reloadPageAndRunAllTests) {
```

Comment:
again +1 for an object here, basically these are initial setup options when abstracted one level I think.